### ISSUE TITLE
Update README to reflect new features

### ISSUE THREAD

**Issue by john-g-g** [steemit/jussi#103]

**Comment by bnchdrff** [comment]
here's how i disabled upstream tests -- you need to do this if you're providing an upstream on a non-https or ws port:

```diff
diff --git a/Dockerfile b/Dockerfile
index c6d15db..2ad9374 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -21,7 +21,7 @@ ENV APP_CMD jussi.serve
 ENV JUSSI_SERVER_HOST 0.0.0.0
 ENV JUSSI_SERVER_PORT 9000
 ENV JUSSI_UPSTREAM_CONFIG_FILE ${APP_ROOT}/PROD_UPSTREAM_CONFIG.json
-ENV JUSSI_TEST_UPSTREAM_URLS True
+ENV JUSSI_TEST_UPSTREAM_URLS False
 
 # all nginx env vars must also be changed in service/nginx/nginx.conf
 ENV NGINX_SERVER_PORT 8080
```

might as well add how you build & run:

```
docker build -t="$USER/jussi:$(git rev-parse --abbrev-ref HEAD)" .
docker run -itp 9000:8080 "$USER/jussi:$(git rev-parse --abbrev-ref HEAD)"
```

**Comment by jredbeard** [comment]
Resolved in #219
